app/commands.py

```diff
@@ -2267,7 +2267,10 @@ async def pool_info(ctx: Context) -> Optional[str]:
     datetime_fmt = f"Created at {_time} on {_date}"
     l = [f"{pool.id}. {pool.name}, by {pool.created_by} | {datetime_fmt}."]
 
-    for (mods, slot), bmap in pool.maps.items():
+    for (mods, slot), bmap in sorted(
+        pool.maps.items(),
+        key=lambda x: (Mods.to_string(x[0][0]), x[0][1]),
```

Comment:
i wonder if this would benefit from casefold.. i think we should normalize all mod inputs (maybe we already do) to solve any inconsistencies in casing